examples/Python/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -30,17 +30,25 @@
 # Select only key-values that start with 'testapp_settings_' and trim the prefix
 selects = SettingSelector(key_filter="testapp_settings_*")
 selects_secret = SettingSelector(key_filter="secret_key")
-AZURE_APPCONFIGURATION = load(endpoint=ENDPOINT,
-              keyvault_credential=credential,
-              credential=credential,
-              selects=[selects, selects_secret],
-              trim_prefixes=["testapp_settings_"],
-              refresh_on=[SentinelKey("sentinel")]
-              )
+
+def callback():
+    # Update Django settings with the app configuration key-values
+    CONFIG.update(AZURE_APP_CONFIG)
```

Comment:
Should this be inline on line 44? 

`on_refresh_success=lambda: CONFIG.update(AZURE_APP_CONFIG)`

Reply:
@albertofori, I was thinking about not doing that as this keeps it simple. And shows you can do what ever you want. Plus the comment is easy to map to the callback.